win32ss/gdi/ntgdi/freetype.c

```diff
@@ -5894,7 +5864,7 @@ ftGdiGetTextWidth(
     LONG lfHeight,
     UINT fuOptions,
     FT_Render_Mode RenderMode,
-    PMATRIX pmxWorldToDevice,
+    const FT_Matrix *pmat,
```

Comment:
Did you change the parameter name because it can be a matrix for anything? Otherwise, the "WorldToDevice" name seemed to be more descriptive of its purpose.

Reply:
Renamed as `pmatTransform`.

Reply:
This matrix will be used for any transformation in future.